core/statistics/softwareVersion/softwareVersion_test.go

```diff
@@ -0,0 +1,37 @@
+package softwareVersion
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/consensus/mock"
```

Comment:
why you use the AppStatusHandlerMock from consensus package ? there is a stub for this component in the core package

Reply:
done.